### ISSUE TITLE
Training Bokeh Problem

### ISSUE THREAD

**Issue by SueeH** [aiff22/PyNET-PyTorch#24]
In the project, " Compared to the original [TensorFlow model](https://github.com/aiff22/PyNET), this implementation contains three major modifications:
Instance normalization is used in PyNET's level 1.
Transposed convolutional layers are replaced with upsampling convolution.
Modified weight coefficients of the loss functions. "

So if I want train bokeh with ebb dataset, I  need 
1. not used instance normalization in level ?
2. upsampling convolution still used?
3. what's weight coefficients of the loss functions?
It just confused me ！
